include/aikido/util/ExecutorMultiplexer.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef AIKIDO_UTIL_EXECUTORMULTIPLEXER_HPP_
+#define AIKIDO_UTIL_EXECUTORMULTIPLEXER_HPP_
+
+#include <functional>
+#include <mutex>
+#include <vector>
+
+namespace aikido {
+namespace util {
+
+/// ExecutorMultiplexer stores callbacks and calls them in order of they added.
```

Comment:
The fact that callbacks are called in the order they are added isn't important. My original intent for implementing this class was to allow one `ExecutorThread` to call multiple executors. The purpose of `ExecutorThread` is to periodically call "spin" functions, like those in `RosJointStateClient` and `RosTrajectoryExecutor`.

I'd like to capture this "big picture" idea here. Here's my attempt at a new docstring:

> Combine multiple executors (i.e. no argument callbacks) into one executor.
>
> This helper class allows one `ExecutorThread to call multiple executors by sequentially calling the callbacks added to this class.

It is not critical, but I would slightly prefer to remove all of the comments that specify the order the callbacks will be called in. The fact that they are called in order is an implementation detail that is subject to change - and may very well need to change if we add a `removeCallback` helper function to this class.